codec/src/main/java/io/netty5/handler/codec/LengthFieldBasedFrameDecoderForBuffer.java

```diff
@@ -0,0 +1,475 @@
+/*
+ * Copyright 2022 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty5.handler.codec;
+
+import io.netty5.buffer.api.Buffer;
+import io.netty5.channel.ChannelHandlerContext;
+
+import java.nio.ByteOrder;
+
+import static io.netty5.util.internal.ObjectUtil.*;
+import static java.util.Objects.*;
```

Comment:
Wildcards are not allowed

Reply:
Fixed it. However this seems to be configured in the code style. Maybe change it there too?

![screenshot](https://user-images.githubusercontent.com/17328331/162619011-89955a79-cc49-4ffb-bb3c-e111d6b0d671.png)

Reply:
That's an upper limit to prevent IntelliJ from using wildcards.

Reply:
Yes, but it's `1` for _static_ imports, so IntelliJ will always use a wildcard for them. If it was set to `99`, it would never use wildcard imports.

Reply:
Created a PR to change the code style setting https://github.com/netty/netty-website/pull/12